admin/partials/metabox-publish.php

```diff
@@ -17,7 +17,7 @@
 		?>
 	</div>
 	<?php return; ?>
-<?php endif; ?>
+<?php endif;  // phpcs:ignore Squiz.PHP.NonExecutableCode.Unreachable ?>
```

Comment:
What specifically was causing this?

Reply:
```
FILE: admin/partials/metabox-publish.php
-----------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------
 20 | WARNING | Code after the RETURN statement on line 19 cannot be executed
    |         | (Squiz.PHP.NonExecutableCode.Unreachable)
-----------------------------------------------------------------------------------------------------
```

Reply:
Seems like a PHPCS bug, no?

Reply:
Yeah, it does. I wonder if we did this instead if it would still trigger:

```php
if ( $condition) {
    ?>
    <html here>
    <?php
    return;
}
```